src/plugins/telemetry/server/routes/telemetry_opt_in_stats.ts

```diff
@@ -35,7 +35,7 @@ export async function sendTelemetryOptInStatus(
     channelName: 'optInStatus',
   });
 
-  const optInStatusPayload: UnencryptedTelemetryPayload =
+  const optInStatusPayload: UnencryptedTelemetryPayload | EncryptedTelemetryPayload =
```

Comment:
It could also return `EncryptedTelemetryPayload` (actually, attending to the usage, it's the most likely output) 😇